doc/concepts/pipeline-concepts/pipeline/cron.md

```diff
@@ -0,0 +1,69 @@
+# Cron Pipeline
+
+Pachyderm triggers pipelines when new changes appear in the input repository.
+However, if a pipeline consumes data from sources outside of Pachyderm,
```

Comment:
> However, if a pipeline consumes data from sources outside of Pachyderm,
it cannot use Pachyderm's triggering mechanism to process updates from
those sources. A standard pipeline with a PFS input might not satisfy
the requirements of the following tasks

However, you want to trigger a pipeline based on time instead waiting on the arrival of input data. You can schedule such pipelines to run periodically by using the Pachyderm's built-in `cron` input type.

Common uses of cron inputs include: